solutions/devsprint-eduardo-bocato-1/SpaceApp/Modules/Features/Details/Scenes/LaunchDetails/Views/LaunchDetailsHeaderView.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  LaunchDetailsHeaderView.swift
+//  SpaceApp
+//
+//  Created by vinicius.carvalho on 15/03/22.
+//
+
+import SwiftUI
+
+struct LaunchDetailsHeaderView: View {
+    struct Model {
+        let image: UIImage
```

Comment:
**[CHANGE REQUEST]**
This is UIKit, please avoid.
Also, a model should hold data... UIImage is a UI thing.